src/data/markdown/docs/01 guides/02 Using k6/14 Scenarios/01 Executors/01 shared-iterations.md

```diff
@@ -0,0 +1,63 @@
+---
+title: 'Shared iterations'
+excerpt: ''
+---
+
+> ### 🎉 New in v0.27.0
+>
+> This feature is new as of version 0.27.0. Usage of this feature is optional and for the vast majority,
+> existing scripts and configurations will continue to work as before. For a list of breaking changes,
+> see the [k6 v0.27.0 release notes](https://github.com/loadimpact/k6/releases/tag/v0.27.0)).
```

Comment:
Agree with Pepe that we shouldn't repeat this header everywhere and only show it on the main Scenarios page.